On The Map/Reachability.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Reachability.swift
+//  On The Map
+//
+//  Created by Avinash Mudivedu on 4/7/16.
+//  Copyright © 2016 Avinash Mudivedu. All rights reserved.
+//
+//  SOURCE: http://www.brianjcoleman.com/tutorial-check-for-internet-connection-in-swift/
+
+import SystemConfiguration
+
+public class Reachability {
```

Comment:
BTW, this is carryover from Objective C and I don't know if it applies to Swift, but I figure folks just follow it any way.
It comes from https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Conventions/Conventions.html :
> Your own classes should use three letter prefixes. These might relate to a combination of your company name and your app name, or even a specific component within your app. As an example, if your company were called Whispering Oak, and you were developing a game called Zebra Surprise, you might choose WZS or WOZ as your class prefix.